src/App.js

```diff
@@ -31,6 +34,7 @@ const SearchResults = React.lazy(() => import('./pages/SearchResults'));
 const App = () => {
   return (
     <Router>
+      <ToastContainer />
```

Comment:
Will this new ToastContainer component post the notification on the top of the page just like you showed in our meeting?

Reply:
Yes! exactly like in our meeting